bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -585,7 +585,7 @@
    // ############################################# TRAINING #############################################
 
    /**
-    * Naming variants from Jesús Pérez
+    * Naming variants from JesÃºs PÃ©rez
```

Comment:
same here

Reply:
@FJBDev 
thanks, I made the correction. it seems somehow/somewhere I endup with some character encodin issue.

Reply:
That happens to me too at times. It must be buried into a deep setting in eclipse because I dont remember how I fixed it permanently